bin/func_tests.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+
+echo "Running functional tests..."
+coverage run -m pytest src\tests\functional\test_main.py --token %CBC_AUTH_TOKEN%
```

Comment:
Any reason you want to run the functional tests inside of coverage tool instead of just running pytest? I am not sure we will get useful coverage numbers from just the functional tests

Reply:
That's a good point for all of the places we initiate functional testing, where just pytest would suffice

Reply:
I was following the pattern that was used in the other scripts.  I can clean that up.